public/index.html

```diff
@@ -0,0 +1,370 @@
+                    <tr>
+                        <td>7 Projects</td>
+                    </tr>
+                    <tr>
+                        <td>5 Users</td>
+                    </tr>
+                    <tr>
+                        <td>10 Gb user</td>
+                    </tr>
+                    <tr>
+                        <td>5 Calendar </td>
+                    </tr>
+                    <tr class="table-foot">
+                        <td>
+                            <button type="button" class="button-signup">SIGN UP</button>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+            <table class="flex-item-3">
+                <thead>
+                    <tr class="table-head">
+                        <th>PROFESSIONAL</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td class="table-price">$25.99/MONTH</td>
+                    </tr>
+                    <tr>
+                        <td>15 Projects</td>
+                    </tr>
+                    <tr>
+                        <td>50 Users</td>
+                    </tr>
+                    <tr>
+                        <td>100 Gb user</td>
+                    </tr>
+                    <tr>
+                        <td>25 Calendar </td>
+                    </tr>
+                    <tr class="table-foot">
+                        <td>
+                            <button type="button" class="button-signup">SIGN UP</button>
+                        </td>
+                    </tr>
+                    <tbody>
+            </table>
+            <table class="flex-item-3">
+                <thead>
+                    <tr class="table-head">
+                        <th>PREMIUM</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td class="table-price">$49.99/MONTH</td>
+                    </tr>
+                    <tr>
+                        <td>UNLIMITED Projects</td>
+                    </tr>
+                    <tr>
+                        <td>UNLIMITED Users</td>
+                    </tr>
+                    <tr>
+                        <td>1000 Gb user</td>
+                    </tr>
+                    <tr>
+                        <td>UNLIMITED Calendar </td>
+                    </tr>
+                    <tr class="table-foot">
+                        <td>
+                            <button type="button" class="button-signup">SIGN UP</button>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        </section>
+    </section>
+    <section class="section-white">
+        <object class="clip clip-orange" type="image/svg+xml" data="./imgs/cliporange.svg"></object>
+        <a id="portfolio"></a>
+        <h1>PORTFOLIO</h1>
+        <div class="carousel_portfolio">
```

Comment:
Completely unnecessary element. No styling, and no semantic benefit.

Perhaps you're planning ahead for when this is a functional carousel?